src/main/kotlin/org/jetbrains/bio/genome/Genome.kt

```diff
@@ -645,6 +645,7 @@ fun String.toStrand() = single().toStrand()
 fun Char.toStrand() = when (this) {
     '+' -> Strand.PLUS
     '-' -> Strand.MINUS
+    'C' -> Strand.MINUS
```

Comment:
Can you please remove this option or add this super specific hs1 option processing to hs1 parsing?
Another possible solution is to add explicit comments on why we need this exception.

Reply:
I've moved this option to AnnotationsHs1.kt